microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -177,20 +226,74 @@ def _install(self) -> bool:
             node.get_pure_path("~/.bashrc"),
             append=True,
         )
+
         return True
 
-    def _install_dependencies(self) -> None:
-        node = self.node
-        cwd = node.working_path
-        lspci = node.tools[Lspci]
-        device_list = lspci.get_device_list()
-        self.is_connect_x3 = any(
-            ["ConnectX-3" in dev.device_info for dev in device_list]
-        )
+    def set_version_info_from_source_install(
+        self, branch_identifier: str, matcher: Pattern[str]
+    ) -> None:
+        match = matcher.search(branch_identifier)
+        if not match or not match.group("major") or not match.group("minor"):
+            fail(
+                f"Could not determine dpdk version info from '{self._dpdk_source}'"
+                f" with id: '{branch_identifier}' using regex: '{matcher.pattern}'"
+            )
+        else:
+            major, minor = map(int, [match.group("major"), match.group("minor")])
+            self._dpdk_version_info = VersionInfo(major, minor)
+
+    def _load_drivers_for_dpdk(self) -> None:
+        self.node.log.info("Loading drivers for infiniband, rdma, and mellanox hw...")
         if self.is_connect_x3:
             mellanox_drivers = "mlx4_core mlx4_ib"
         else:
             mellanox_drivers = "mlx5_core mlx5_ib"
+        if isinstance(self.node.os, Ubuntu):
+            self.node.execute(
+                "modprobe -a rdma_cm rdma_ucm ",
+                cwd=self.node.working_path,
+                expected_exit_code=0,
+                expected_exit_code_failure_message=(
+                    "Could not load rdma cm/ucm drivers"
```

Comment:
If the `stdout` is helpful, it can be included here. But if it's too long, consider extracting the key lines.

Reply:
I don't think this command dumps anything, if it fails it just returns an error code on failure.

Reply:
BTW, there is a tool `modprobe` already, please reuse it. It uses `-v` parameter to output.

Reply:
Can you change it to use modprobe tool?